### PR TITLE
[config] Add subcommand to configure the status of auto-restart feature for each container

### PR DESCRIPTION
**Pull request by yozhao101**
- What I did
Since we introduced the auto-restart feature for each container, we need add a config subcommand to change the current status (enabled/disabled) of auto-restart feature for a specific container.

- How I did it
We define a function named autorestart to change the status of a container. This function will accept two parameters which are the container name and new status. If one of parameters is not specified, this function will show the information and then exit.

- How to verify it
We can run the command **config container feature autorestart container_name autorestart_status**.

### CODE REVIEW

**Comment by yozhao101** [comment]
@jleveque Sure.

File: config/main.py [hunk 1]
```diff
@@ -2468,5 +2468,41 @@ def feature_status(name, state):
 
     config_db.mod_entry('FEATURE', name, {'status': state})
 
+#
+# 'container' group ('config container ...')
+#
+@config.group(name='container', invoke_with_command=False)
+def container():
+    """Config container"""
```
Comment (jleveque) [inline config/main.py]: Change help string to `Modify configuration of containers`

Also update this in Command reference PR.

File: config/main.py [hunk 2]
```diff
@@ -2468,5 +2468,41 @@ def feature_status(name, state):
...
+    pass
+
+#
+# 'feature' group ('config container feature ...')
+#
+@container.group(name='feature', invoke_with_command=False)
+def feature():
+    """Config container feature"""
```
Comment (jleveque) [inline config/main.py]: Change help string to `Modify configuration of container features`

Also update this in Command reference PR.

File: config/main.py (hunk 1 above)
Comment (yozhao101) [reply config/main.py]: Fixed.

File: config/main.py (hunk 2 above)
Comment (yozhao101) [reply config/main.py]: Fixed.

**Comment by jleveque** [comment]
Retest this please

**Pull request merged**